contributors/devel/coding-conventions.md

```diff
@@ -75,7 +75,7 @@ following Go conventions - `stateLock`, `mapLock` etc.
 tests
 
   - Table-driven tests are preferred for testing multiple scenarios/inputs; for
-example, see [TestNamespaceAuthorization](../../test/integration/auth/auth_test.go)
+example, see [TestNamespaceAuthorization](https://github.com/kubernetes/kubernetes/blob/master/test/integration/auth/auth_test.go)
```

Comment:
@cblecker - I'm fine with using the path that you mention, it resolves to the URL I specified. Why do you prefer this path?

Note: the path I've used is referenced in a few other spots in the community repo.